bubblescan-client/src/components/FileUploadComponent.tsx

```diff
@@ -7,14 +7,20 @@ function FileUploadComponent() {
   const [fileId, setFileId] = useState<string>("");
   const [loading, setLoading] = useState<boolean>(false);
 
+  // Handle file input change
   const handleFileChange = (event: React.ChangeEvent<HTMLInputElement>) => {
     const selectedFile = event.target.files?.[0];
-    if (selectedFile && selectedFile.type === "application/pdf") {
+    if (
+      selectedFile &&
+      (selectedFile.type === "application/pdf" ||
+        selectedFile.type.startsWith("image/"))
```

Comment:
Nice job changing the types of images allowed. Hopefully the rest of the program accepts these files.